CHANGELOG.md

```diff
@@ -4,6 +4,29 @@ Notable changes to Draft.js will be documented in this file.
 
 Changes to `src` are live in production on facebook.com at the time of release.
 
+## 0.10.2
+
+### Added
+* Change aria-owns to aria-controls in draft.js ([@jessebeach](https://github.com/jessebeach) in [7f0cab28](https://github.com/facebook/draft-js/commit/7f0cab28386ec4bde8ec6289377bff9e53cd019b))
+  * Adds support for `ariaControls` prop
+
+### Changed
+* Change aria-owns to aria-controls in draft.js ([@jessebeach](https://github.com/jessebeach) in [7f0cab28](https://github.com/facebook/draft-js/commit/7f0cab28386ec4bde8ec6289377bff9e53cd019b))
```

Comment:
Dupe line?

Reply:
I thought it qualified as a 'changed' as well as an 'added' item, but probably could just list it under one or the other.

Reply:
I think it is OK to have under both if the wording is different but it's confusing if it's identical.